packages/cli/src/commands/server/runServer.ts

```diff
@@ -108,6 +120,12 @@ async function runServer(_argv: Array<string>, ctx: Config, args: Args) {
     '/debugger-proxy',
   );
   const ms = messageSocket.attachToServer(serverInstance, '/message');
+  eventsSocket = eventsSocketModule.attachToServer(
+    serverInstance,
+    '/events',
+    ms,
+  );
+  logger.log('Created events socket');
```

Comment:
Do we need this console log here?

Reply:
The way Flipper works, is that it scans the system for all open terminals, and parses their output stream. To find out which one belongs to Metro, it need to be able to recognize it, so this logging statement is secretly used as a marker so that we know that we are looking at the metro output stream.

Reply:
Kidding, will remove.